### PR TITLE
Task 5

### PR DESCRIPTION
**Pull request by DddimaChe**

### CODE REVIEW

**Comment by DddimaChe** [comment]
FE: https://dxt9a3tcwgmqz.cloudfront.net/
URL: https://5rrg82irld.execute-api.eu-west-1.amazonaws.com/dev/import
Params: name

**Comment by tapion** [comment]
Great job!

File: import-service/src/functions/importFileParser/index.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+const headers = {
+    "Access-Control-Allow-Origin": "*",
+}
+
+const sendFileToBucket = async (records: any) => {
+    const promises = records.map(async (record) => {
+        const bucket = record.s3.bucket.name;
+        const key = record.s3.object.key;
+        const s3 = new AWS.S3({ region: REGION });
```
Comment (VitaliiKopylov) [inline import-service/src/functions/importFileParser/index.ts:16]: Maybe you can reuse `s3` instance from `importFileParser` function

**Comment by VitaliiKopylov** [comment]
All tasks are completed.
**Total score: 8**

**Pull request merged**